src/azure-cli/azure/cli/command_modules/redis/commands.py

```diff
@@ -28,26 +28,26 @@ def load_command_table(self, _):
 
     with self.command_group('redis', redis_sdk) as g:
         g.custom_command('create', 'cli_redis_create', client_factory=cf_redis)
-        g.command('delete', 'delete', confirmation=True)
+        g.command('delete', 'begin_delete', confirmation=True)
         g.custom_command('export', 'cli_redis_export')
-        g.command('force-reboot', 'force_reboot')
-        g.command('import-method', 'import_data', deprecate_info=g.deprecate(redirect='redis import', hide=True))
-        g.command('import', 'import_data')
+        g.custom_command('force-reboot', 'cli_redis_force_reboot', client_factory=cf_redis)
```

Comment:
why do this? We already defined the command groups in the beginning so that we dont have to add factory prefix to each commands. Why are we changing this?

Reply:
Modified